src/App/App.js

```diff
@@ -11,29 +11,41 @@ class App extends Component {
     this.state = {
       movies: [],
       error: false,
+      filteredMovies: []
     }
   }
 
   componentDidMount() {
     grabAPI()
       .then(data => {
         let cleanData = data.movies.map(movie=> {
-           return {
+          return {
             ...movie, 
             average_rating: movie.average_rating.toFixed(1)
           }
         })
-        // console.log(cleanData)
-        this.setState({ ...this.state, movies: cleanData })
+        this.setState({ ...this.state, movies: cleanData, filteredMovies: cleanData })
+
       })
       .catch(() => this.setState({ ...this.state, error: true }))
   }
 
+  newFilteredMovies = (e) => {
```

Comment:
Great job on figuring out this functionality on how to search the filtered movies!